daml-lf/spec/daml-lf-1.rst

```diff
@@ -2394,6 +2522,60 @@ exact output.
         ⇓
       Ok ⟨ f₁= v₁, …, fᵢ= vᵢ', …, fₙ= vₙ ⟩
 
+      e  ⇓  Err v
+    —————————————————————————————————————————————————————————————————————— EvExpErrorErr
+      'ERROR' @τ e  ⇓  Err v
+
+      e  ⇓  Ok v
+    —————————————————————————————————————————————————————————————————————— EvExpError
+      'ERROR' @τ e
+        ⇓
+      Err ('make_any_exception' @'GeneralError' v ('MAKE_GENERAL_ERROR' v))
```

Comment:
I will drop this rule and add in the definition of 'ERROR' down that it is an alias for throw similarly we done for `GREATER_EQ`
```'ERROR' ≡  Λ α : ⋆. λ x : 'Text'. THROW @t x (ToAnyException @GeneralError (MAKE_GENERAL_ERROR x)```

Reply:
should be

`'ERROR' ≡ Λ α : ⋆. λ x : 'Text'. 'THROW' @α ('make_any_exception' @'GeneralError'  x ('MAKE_GENERAL_ERROR' x))`

Reply:
down meaning in the `Built-in functions` section.